public/api/conf/3.0/application.yaml

```diff
@@ -126,6 +131,22 @@ paths:
                     name: CustomsDeclarationsresponse400
                     attribute: false
                     wrapped: false
+        '403':
+          description: ''
+          headers: {}
+          content:
+            application/xml:
+              examples:
+                objectExample:
+                  $ref: '#/components/examples/PayloadForbiddenExample'
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/CustomsDeclarationsresponse400'

```

Comment:
Inconsistent indentation.

Reply:
indentation is the same ad the 400 errors

Reply:
done